docs/guidelines/git-github.md

```diff
@@ -0,0 +1,306 @@
+# Git and Github
+
+This guideline contains some basic information on configuration of Git and user profiles on github.com. Our perspective would be security and privacy. The guideline is by no means exhaustive, it's more an introduction to basic config and the correlation between Git and github.com
+
+!!! tip "Git vs Github.com"
+
+    Git is a distributed version control system for tracking changes in source code, while GitHub is a platform that hosts Git repositories online. GitHub builds upon Git, offering a centralized place for developers to share and work on code together.
+
+
+!!! Info "The SCM Policy"
+
+    [The Equinor Developer Portal](https://developer.equinor.com/) contain our [Source Code Management System Policy](https://developer.equinor.com/governance/scm-policy/). Please make sure you are familiar with the content.
+
+## TLDR;
+
+- Configure your local Git with proper name and email. Use your full name with the profile. Use privacy options for hiding your github.com primary email.
+- For authentication and communication with github.com, use Git with SSH and passphrase protected private keys.
+- Use separate SSH keys for different client devices.
+- Book a yearly re-occurring event with yourself to rotate SSH keys.
+
+## Git
+
+For this guideline we use Git from the command line. Be aware, there are many tools that hide the internal mechanics of Git within the tool. For these tools most settings are defined within the tool itself.
+
+We assume that `git` and `ssh` are installed on your system. We do not cover the installation process besides mentioning the fact that git and ssh like any other piece of software must be kept up-to-date.
```

Comment:
Link to https://git-scm.com/downloads to install git?

Reply:
I'll create a section at the end with external resources and include the link in that section. At this stage we should not include "how" to install information as it's quite different from system to system .....